src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -281,6 +281,15 @@ static RequestExecutor()
             {
                 using (var handler = new HttpClientHandler())
                     handler.ServerCertificateCustomValidationCallback += OnServerCertificateCustomValidationCallback;
+
+                string explicitlyTrustedCerts = Environment.GetEnvironmentVariable("Raven_ExplicitlyTrustedServerCertificateThumbprints") ?? 
```

Comment:
do we want to use configuration mechanism here? So we can pick it up from settings.json or from env variables?

Reply:
Note that this is happening on the _client_ side, we don't have configuration mechanism here.
I also assume that the user may not have setup anything to make it work, so we can't rely on code configuration.

Assume that you have a random RavenDB client using this, and we need to support a certificate explicitly. And immediately. Env variable is the easiest way to do that without changing any code.